src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -131,7 +131,12 @@
     parameters:
         - name: --image
           type: string
-          short-summary: The name of the operating system image as a URN alias, URN, or URI.
+          short-summary: >
+            The name of the operating system image as a URN alias, URN, custom image name or ID, or VHD blob URI.
+            This parameter is required unless using `--attach-os-disk.` Valid URN format: "Publisher:Offer:Sku:Version".
```

Comment:
again, the URN is not the only format accepted

Reply:
Yes I agree, but I added the other formats in the help text. Do you want me to rephrase the short-summary help text?